framework/src/st_things/things_stack/utils/things_wait_handler.c

```diff
@@ -21,14 +21,13 @@
 #include <stdlib.h>
 #include <pthread.h>
 
-#include "things_malloc.h"
-#include "things_logger.h"
-#include "things_wait_handler.h"
-
 #include "things_thread.h"
 #ifdef __ST_THINGS_RTOS__
```

Comment:
Do we need definition ?

Reply:
Yes. It makes difference when create pthread.